app/code/Magento/CmsGraphQl/Model/Resolver/DataProvider/Page.php

```diff
@@ -26,6 +26,11 @@ class Page
      * @var PageRepositoryInterface
      */
     private $pageRepository;
+    
+    /**
+     * Page Url Key
+     */
+    const URL_KEY = 'url_key';
```

Comment:
Unfortunately, in this case, it makes no sense to use a constant. This value is used in just one place.
If you would like to contribute, please ping me(@vnayda ) in Slack channel and I will assist you with a suitable task